scenario_editor/src/Home/Home.js

```diff
@@ -3,18 +3,26 @@ import './Home.css';
 import Nav from '../Components/Nav'
 import Button from 'react-bootstrap/Button';
 import {Link} from 'react-router-dom';
+import Jumbotron from 'react-bootstrap/Jumbotron'
+import ButtonGroup from 'react-bootstrap/ButtonGroup'
 
 function Home() {
   return (
     <div> 
       <Nav/>
```

Comment:
Rendering nav in each component is unnecessary. Just render it once in App.js. If you don't want it to show if a user isn't logged in, do something like 
`userIsLoggedIn && <Nav/>`
where userIsLoggedIn just checks if the user is logged in, however you handle that (cookies, local storage, etc)